CODE_OF_CONDUCT.md

```diff
@@ -1,10 +1,115 @@
+- Likewise any spamming, trolling, flaming, baiting or other attention-stealing behavior is not welcome.
+
+### Moderation
+
+These are the policies for upholding our community’s standards of conduct. If
+you feel that a thread needs moderation, please
+[contact the moderation team](#contact).
+
+- Remarks that violate the above code of conduct, including hateful, hurtful, oppressive, or exclusionary remarks, are not allowed. (Cursing is allowed, but never targeting another user, and never in a hateful or aggressive manner.)
+- Moderators will warn users who make remarks inconsistent with the above code of conduct.
+- If the warning is unheeded, the user will be “kicked,” i.e., kicked out of the communication channel to cool off.
+- If the user comes back and continues to make trouble, they will be banned, i.e., indefinitely excluded.
+- Moderators may choose at their discretion to un-ban the user if it was a first offense and they if they make suitable amends with the offended party.
+- If you think a moderator action is unjustified, please take it up with that moderator, or with a different moderator, in private. Complaints about moderation in-channel are not allowed.
+- Moderators are held to a higher standard than other community members. If a moderator acts inappropriately, they should expect less leeway than others.
+
+In the Play Framework community we strive to go the extra step to look out for each
+other. Don’t just aim to be technically unimpeachable; try to be your best self.
+In particular, avoid exacerbating offensive or sensitive issues, particularly if
+they’re off-topic; this all too often leads to unnecessary fights, hurt
+feelings, and damaged trust; worse, it can drive people away from the community
+entirely.
+
+If someone takes issue with something you said or did, resist the urge to be
+defensive. Rather, stop the offending behavior, apologize, and be sensitive
+thereafter. Even if you feel you were misinterpreted or unfairly accused,
+chances are good there was something you could’ve communicated better — remember
+that it’s your responsibility to make your fellow community members comfortable.
+We are all here first and foremost because we want to talk about cool
+technology, and everyone wants to get along in doing so. People are generally
+eager to assume good intent and forgive.
+
+### Domain
+
+The enforcement policies listed above apply to all official Play Framework channels, including:
+
+* GitHub discussion forums
+* Chat rooms under the "Play Framework" Discord server
+* Mailing lists
+* GitHub repositories under the [`playframework` organization](https://github.com/playframework)
+* Play Framework video streams ([Twitch](https://www.twitch.tv/playframework) and [YouTube](https://www.youtube.com/channel/UCRp6QDm5SDjbIuisUpxV9cg))
```

Comment:
No possible yet: "_Handles will be rolling out soon. We’ll notify you when you can choose yours._"
Also see https://support.google.com/youtube/thread/183284252
> Q: When can I choose a handle?
> ...we're rolling them out gradually... Everyone will have the option to choose a handle by November 14 at the latest.

Reply:
Hmmm, I'm lucky 😄 I already received a message and chose my handle 😄

Reply:
We now have: https://youtube.com/@play-framework
The problem is there is a very old channel from 2009 which has the https://www.youtube.com/@playframework name. This is because they have the name https://www.youtube.com/user/playframework and now Google automatically gives the handle the name of the user. (the same automatic mechanism happened to my personal youtube account, I didn't have to choose my handle myself)
Would be interesting to find out who owns the channel https://www.youtube.com/@playframework .... It's not used so I think maybe someone from the very very old original Play team...